thoughtbot-about.md

```diff
@@ -0,0 +1,35 @@
+# thoughtbot.social
+
+## Code of Conduct
+
+In order to foster an inclusive, kind, harassment-free, and cooperative community, thoughtbot enforces this code of conduct on this social network.
+
+### Summary
+
+Harassment in code and discussion or violation of physical boundaries is completely unacceptable anywhere in thoughtbot’s social network, project codebases, issue trackers, chatrooms, mailing lists, meetups, and other events. The core team uses their discresion to choose whether to warn violators. Repeat violations will result in being blocked or banned by the core team at or before the 3rd violation.
+
+Existing employees are expected to live up to the same standards and will be handled by our internal processes.
+
+### In detail
+
+Harassment includes offensive verbal comments related to gender identity, gender expression, sexual orientation, disability, physical appearance, body size, race, religion, sexual images, deliberate intimidation, stalking, sustained disruption, and unwelcome sexual attention.
+
+Individuals asked to stop any harassing behavior are expected to comply immediately.
+
+Existing employees are also subject to the anti-harassment policy. Violation will be handled internally in our usual manner.
+
+If anyone engages in harassing behavior, including maintainers, we may take appropriate action, up to and including warning the offender, escalation to their admin, and defederation.
```

Comment:
This might be stronger. "may" maybe leaves too much ambiguity?

```suggestion
If anyone engages in harassing behavior, including maintainers, we will take appropriate action, up to and including warning the offender, escalation to their admin, and defederation.
```